check_clang_format.sh

```diff
@@ -0,0 +1,26 @@
+# Install Dependencies
+travis_run sudo apt-get install clang-format-3.6
+
+# Download config script
+travis_run git clone https://github.com/davetcoleman/roscpp_code_format
+travis_run mv roscpp_code_format/.clang-format .
+
+# Change to source directory
+travis_run cd $CI_SOURCE_PATH
+travis_run ls -la
+
+# Run clang-format
+echo "Running clang-format"
+find . -name '*.h' -or -name '*.hpp' -or -name '*.cpp' | xargs clang-format-3.6 -i -style=file
+
+echo "Showing results"
+travis_run git --no-pager diff
+
+# Make sure no changes have occured in repo
+if ! git diff-index --quiet HEAD --; then
+    # changes
+    echo "clang-format test failed: uncommitted changes detected in repo after running clang-format"
```

Comment:
Consider an error message like: "clang-format test failed: changes required to comply to formatting rules"